components/datetime/src/error.rs

```diff
@@ -68,6 +68,15 @@ pub enum DateTimeFormatterError {
         "TypedDateTimeFormatter<{0}> was given a locale asking for incompatible calendar u-ca-{1}"
     )]
     MismatchedCalendarLocale(&'static str, TinyStr16),
+    /// TODO
+    #[displaydoc("TODO")]
```

Comment:
@zbraniecki

Reply:
The displaydoc is provided in the `expect` that you replaced with this errors:
 - `Expect date symbols to be present`
 - `Expect time symbols to be present`
 - `ordinal_rules must be set with PatternPlurals::MultipleVariants`

Reply:
I didn't think those were very consistent with the other error messages